vecs.cc

```diff
@@ -145,10 +140,9 @@ struct Vecs::Impl {
   GarminFitFormat format_fit_fmt;
   GeoJsonFormat geojson_fmt;
   GlobalsatSportFormat globalsat_sport_fmt;
-#endif // MAXIMAL_ENABLED
+  QstarzBL1000Format qstarz_bl_1000_fmt;
```

Comment:
We only need static instances of formats that aren't dynamic.  The QStarzBL1000 is dynamic, so no static instance is needed.  A format is dynamic if the last entry (factory) is not nullptr, in which case it should be a reference to fmtfactory and the first entry (vec) should be nullptr.

I have lazily been converting format to dynamic.  It involves an evaluation of all the data members, and any lurking static locals, to decide if the format is intentionally storing data between uses, e.g. the gpx read version.  Additionally the Format constructor must be used ("using Format::Format;").  This makes Format::fname available, which can alleviate the need for rd[wr]_init, e.g. geo.